OMEdit/OMEditGUI/Animation/AnimationUtil.h

```diff
@@ -89,8 +89,9 @@ inline bool isCSV(const std::string& fileIn){
  */
 inline std::string assembleXMLFileName(const std::string& modelFile, const std::string& path){
   QString fileName(modelFile.c_str());
-  QRegExp fileTypeRegExp("(\\.fmu|\\.mat|\\.csv|_res.mat|_res.csv)");
-  fileName.remove(fileTypeRegExp);
+  QRegExp fileTypeRegExp("(_res.mat|_res.csv|\\.fmu)");
```

Comment:
So we need another implementation. Strip after last dot. check if result files end with _res and if so, Strip this as well.